### PR TITLE
Make "pihole -d -a" automatically upload the debug log

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following (please check boxes, eg [X]) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:** 10

---

Needed for https://github.com/pi-hole/AdminLTE/pull/399

Can be merged w/o merging https://github.com/pi-hole/AdminLTE/pull/399

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

**Comment by jacobsalmela** [comment]
You mean `pihole -d` right, not `-p`?

**Comment by DL6ER** [comment]
Whoops, Yes. Not `Pebug` but `Debug`

**Comment by dschaper** [comment]
Changed title to "Make Dan finish the code already..."

**Comment by Mcat12** [comment]
Approved

File: advanced/Scripts/piholeDebug.sh [hunk 1]
```diff
@@ -336,7 +342,11 @@ countdown() {
   printf "::: Logging will automatically teminate in ${TIMEOUT} seconds\n"
   while [ $tuvix -ge 1 ]
   do
-    printf ":::\t${tuvix} seconds left. \r"
+    if ! ${automated} ; then
+      printf ":::\t${tuvix} seconds left. \r"
```
Comment (Mcat12) [inline advanced/Scripts/piholeDebug.sh]: Has no newline, causing output like this (captured via web debug):
```
:::	60 seconds left. :::	55 seconds left. :::	50 seconds left.
```

**Comment by dschaper** [comment]
Do not merge until the back end code is there, or we're going to have a bad time.

File: advanced/Scripts/piholeDebug.sh (hunk 1 above)
Comment (dschaper) [inline advanced/Scripts/piholeDebug.sh]: There shouldn't be a new line, it has a carriage return to overwrite the counter on a terminal display.

**Comment by dschaper** [comment]
See #1281 for back and updates.

**Comment by dschaper** [comment]
Does this conflict with #1281 ?

**Comment by DL6ER** [comment]
Implemented by https://github.com/pi-hole/pi-hole/pull/1281

**Pull request closed without merging**